code/modules/recycling/sort_tag.dm

```diff
@@ -0,0 +1,26 @@
+/**
+ * Sorting tag for the sorting machinery.
+ */
+/datum/extension/sorting_tag
+	base_type     = /datum/extension/sorting_tag
+	expected_type = /atom/movable //Both mob and objects can have it
+	var/destination
+	var/tag_icon_state
+	var/tag_x = 0
+	var/tag_y = 0
+
+/datum/extension/sorting_tag/New(datum/holder, var/_destination, var/_icon_state, var/_tag_x = 0, var/_tag_y = 0)
+	. = ..(holder)
```

Comment:
New() has no return value. Don't confuse it with Initialize().
```suggestion
	..(holder)
```

Reply:
Yeah, that's just vscode automatically adding .=..() when autocompleting overriding a proc and me forgetting about it XD.